apps/meeting/src/types/index.ts

```diff
@@ -45,9 +45,22 @@ export enum Layout {
   Featured,
 }
 
-export const BlurValues = {
-  blurDisabled: '0',
-  blur10Percent: '10',
-  blur20Percent: '20',
-  blur40Percent: '40',
+// Different CPU Utilizations percentage options for initializing background blur and replacement processors
+export const VideoFiltersCpuUtilization = {
+  Disabled: '0',
+  CPU10Percent: '10',
+  CPU20Percent: '20',
+  CPU40Percent: '40',
 };
+
+// Video Transform Options 
+export const VIDEO_TRANSFORM_OPTIONS = {
+  none: 'None',
+  blur: 'Background Blur',
+  replacement: 'Background Replacement',
+};
+
+export interface optionsType {
+  label: string,
+  value: string
```

Comment:
optional: Add a trailing comma since you added it to other interfaces above.

Reply:
done